packages/api/src/controllers/user.test.ts

```diff
@@ -221,7 +221,9 @@ describe("controllers/user", () => {
       }
 
       let res = await client.get("/user");
-      expect(res.status).toBe(403);
+      let resJson = await res.json();
+      let condition = Array.isArray(resJson);
+      expect(condition).toBe(false);
```

Comment:
I think `condition` is not a helpful name for a helper variable, as it only creates an indirection that we then need to go back to the original place where the condition was measured to know what it is. I'd either rename this to something clear like `isArray` or `arrayRes` or anything like that, or just skip the helper variable altogether which seems ok imo
```suggestion
      expect(Array.isArray(resJson)).toBe(false);
```